service/src/main/java/uk/gov/hmcts/reform/fpl/utils/IncrementalInteger.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import com.launchdarkly.shaded.javax.annotation.concurrent.NotThreadSafe;
```

Comment:
You are using the right annotation but getting it from the wrong package 😄 
Something like should be all that is needed
```suggestion
import javax.annotation.concurrent.NotThreadSafe;
```

Reply:
Nice spot